index.html

```diff
@@ -33,5 +42,141 @@ <h3>I'm a software Developer, </h3>
+            <li><a href="#">Ruby</a></li>
+          </ul>
+        </div>
+        <button class="project-btn btn-change"><a href="#">See project</a></button>
+      </div>
+      <div class="section-three">
+        <div class="background-holder">
+          <h1>Profesional Art Printing Data</h1>
+          <p>A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+          <ul class="profesional-holder">
+            <li><a href="#">HTML</a></li>
+            <li><a href="#">Bootstrap</a></li>
+            <li><a href="#">Ruby</a></li>
+          </ul>
+        </div>
+        <button class="project-btn btn-change"><a href="#">See project</a></button>
+      </div>
+      <div class="section-three">
+        <div class="background-holder">
+          <h1>Profesional Art Printing Data</h1>
+          <p>A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+          <ul class="profesional-holder">
+            <li><a href="#">HTML</a></li>
+            <li><a href="#">Bootstrap</a></li>
+            <li><a href="#">Ruby</a></li>
+          </ul>
+        </div>
+        <button class="project-btn btn-change"><a href="#">See project</a></button>
+      </div>
+      <div class="section-three">
+        <div class="background-holder">
+          <h1>Profesional Art Printing Data</h1>
+          <p>A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+          <ul class="profesional-holder">
+            <li><a href="#">HTML</a></li>
+            <li><a href="#">Bootstrap</a></li>
+            <li><a href="#">Ruby</a></li>
+          </ul>
+        </div>
+        <button class="project-btn btn-change"><a href="#">See project</a></button>
+      </div>
+    </section>
+    <section class="section-four">
+      <div class="about-me-ctn">
+        <h1>About me</h1>
+        <p>
+          Hello I&apos;m a software developer! I can help you build a product , feature or website Look through some of my work and experience!
+          If you like what you see and have a project you need coded, don&apos;t hesitate to contact me.
+        </p>
+        <button class="btn-change"><a href="#">Get my resume</a></button>
+      </div>
+    </section>
+    <section class="section-five">
+      <img class="divider" src="./final-images/Devider.png" alt="">
+      <div class="language-container">
+         <h2><img src="./final-images/Rectangle 55.png" alt="">Language</h2>
+         <ul>
+          <li>JavaScript</li>
+          <li>Ruby</li>
+          <li>HTML</li>
+          <li>CSS</li>
+         </ul>
+      </div>
+       <div class="frameworks-container">
+          <h2><img src="./final-images/Rectangle 56.png" alt="">Frameworks</h2>
+          <ul>
+           <li>React.js</li>
+           <li>Ruby on Rails</li>
+           <li>RSpec</li>
```

Comment:
- [ ] According to your previous reviewer please ensure that the technology categories and their respective technologies are properly nested within the HTML structure. The technology categories should be represented as `<ul>` elements, with each category as an `<li>`, and the technologies within each category should be represented as nested `<ul>` and `<li>` elements.
- [ ] [See Previouse Review](https://github.com/prince-rutagengwa/mobile-portfolio/pull/2#discussion_r1257265697)